cli/cmd/compose/remove.go

```diff
@@ -0,0 +1,84 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package compose
+
+import (
+	"context"
+
+	"github.com/docker/compose-cli/api/client"
+	"github.com/docker/compose-cli/api/compose"
+	"github.com/docker/compose-cli/api/progress"
+
+	"github.com/spf13/cobra"
+)
+
+type removeOptions struct {
+	*projectOptions
+	force   bool
+	stop    bool
+	volumes bool
+}
+
+func removeCommand(p *projectOptions) *cobra.Command {
+	opts := removeOptions{
+		projectOptions: p,
+	}
+	cmd := &cobra.Command{
+		Use:   "rm [SERVICE...]",
```

Comment:
Could we add `remove` as alias?

Reply:
there's no such thing as a `remove` alias on `docker rm` nor `docker-compose rm`, so to prevent heterogeneous UX I'd prefer we keep such "UX improvement" a separate debate